src/components/ui/OTP/OTP.tsx

```
@@ -0,0 +1,85 @@
+
+  const handleChange = (value: string, index: number) => {
+    const newOtp = [...otp];
+    newOtp[index] = value;
+    setOtp(newOtp);
+
+    if (value.length === 1 && index < length - 1) {
+      inputsRef.current[index + 1]?.focus();
+    }
+
+    if (newOtp.join('').length === length) {
+      onComplete && onComplete(newOtp.join(''));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)